style_desktop.css

```diff
@@ -59,12 +63,17 @@ nav {
   flex-direction: row;
   align-items: center;
   order: 1;
+  padding-top: 0 !important;
 }
 
 .nav_list li {
```

Comment:
- [ ] [REQUIRED] Great implementation so far, however, you still need to ensure that when a user clicks (or taps) any of the mobile menu options (Portfolio, About, Contact), a correct part of the page is displayed.
- [ ] [REQUIRED] When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.
Hint: Add proper links to the following `href`
![Screenshot from 2022-04-05 00-08-12](https://user-images.githubusercontent.com/58692590/161647333-3da28437-f0b3-4620-8a55-3a6234df99c6.png)